server/app/assets/stylesheets/uswds/_uswds-theme.scss

```diff
@@ -0,0 +1,19 @@
+/*
+----------------------------------------
+USWDS with settings overrides
+----------------------------------------
+Uncomment the following lines and add a list of changed settings
+in the form $setting: value,
```

Comment:
suggestion (blocking): nothing is commented below, so this comment seems inappropriate

Reply:
Thanks, will fix these comments.